renovate.json

```diff
@@ -44,5 +44,8 @@
             ],
             "automerge": true
         }
+    ],
+    "schedule": [
+        "on friday"
```

Comment:
let's go for sunday, so, on Monday we get ready to go PR

Reply:
Fixed